### PR TITLE
backward-cpp: add mingw@Linux support

### LINKED ISSUE #6661: [package] backward-cpp/1.5: can't build with mingw
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **backward-cpp/1.5**
  * Operating System+version: **Windows 10**
  * Conan version: **conan 1.38.0**

### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.threads=posix
compiler.version=8.1
os=Windows
os_build=Windows
[options]
[build_requires]
*: mingw-w64/8.1, msys2/cci.latest
[env]
```

### Steps to reproduce (Include if Applicable)
 conan install backward-cpp/1.5@ -pr mingw --build outdated -u

### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

```
conan install backward-cpp/1.5@ -pr mingw --build missing
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.threads=posix
compiler.version=8.1
os=Windows
os_build=Windows
[options]
[build_requires]
*: mingw-w64/8.1, msys2/cci.latest
[env]

You pressed Ctrl+C!
ERROR: Exiting with code: 3
PS C:\Users\AndreiMalashkin\sources\pictofitstudio> conan install backward-cpp/1.5@ -pr mingw --build outdated -u
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.threads=posix
compiler.version=8.1
os=Windows
os_build=Windows
[options]
[build_requires]
*: mingw-w64/8.1, msys2/cci.latest
[env]

mingw-w64/8.1: Package is up to date
msys2/cci.latest: WARN: Current package is newer than remote upstream one
msys2/cci.latest: Package is up to date
Installing package: backward-cpp/1.5
Requirements
    backward-cpp/1.5 from 'artifactory' - Cache
Packages
    backward-cpp/1.5:4b8d03cfef6f209236f18445b773d36681be0d57 - Build
Build requirements
    mingw-w64/8.1 from 'conancenter' - Cache
    msys2/cci.latest from 'conancenter' - Cache
Build requirements packages
    mingw-w64/8.1:6903a9d1b48b06f7fbe0929db654512c77e6cc32 - Cache
    msys2/cci.latest:eee3fba89db6d777329de604625af8c30d46f080 - Cache

Installing (downloading, building) binaries...
mingw-w64/8.1: Already installed!
mingw-w64/8.1: Appending PATH env var with : C:\Users\AndreiMalashkin\.conan\data\mingw-w64\8.1\_\_\package\6903a9d1b48b06f7fbe0929db654512c77e6cc32\bin
msys2/cci.latest: Already installed!
msys2/cci.latest: Creating MSYS_ROOT env var : C:\.conan\ce84ae\1\bin\msys64
msys2/cci.latest: Creating MSYS_BIN env var : C:\.conan\ce84ae\1\bin\msys64\usr\bin
msys2/cci.latest: Appending PATH env var with : C:\.conan\ce84ae\1\bin\msys64\usr\bin
backward-cpp/1.5: Applying build-requirement: mingw-w64/8.1
backward-cpp/1.5: Applying build-requirement: msys2/cci.latest
backward-cpp/1.5: WARN: Build folder is dirty, removing it: C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57
backward-cpp/1.5: Copying sources to build folder
backward-cpp/1.5: Building your package in C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57
backward-cpp/1.5: Generator cmake created conanbuildinfo.cmake
backward-cpp/1.5: Aggregating env generators
backward-cpp/1.5: Calling build()
-- The C compiler identification is GNU 8.1.0
-- The CXX compiler identification is GNU 8.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/bin/gcc.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/bin/g++.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan: Compiler GCC>=5, checking major version 8.1
-- Conan: Checking correct version: 8.1
-- Conan: C++ stdlib: libstdc++11
-- Found Backward: C:/Users/AndreiMalashkin/.conan/data/backward-cpp/1.5/_/_/build/4b8d03cfef6f209236f18445b773d36681be0d57/source_subfolder
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY


-- Build files have been written to: C:/Users/AndreiMalashkin/.conan/data/backward-cpp/1.5/_/_/build/4b8d03cfef6f209236f18445b773d36681be0d57/build_subfolder
[ 66%] [ 66%] Building CXX object source_subfolder/CMakeFiles/backward.dir/backward.cpp.obj
Building CXX object source_subfolder/CMakeFiles/backward_object.dir/backward.cpp.obj
In file included from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:317: warning: "NOMINMAX" redefined
 #define NOMINMAX

In file included from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/x86_64-w64-mingw32/bits/c++config.h:508,
                 from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/utility:68,
                 from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/algorithm:60,
                 from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:73,
                 from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/x86_64-w64-mingw32/bits/os_defines.h:45: note: this is the location of the previous definition
 #define NOMINMAX 1

In file included from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:317: warning: "NOMINMAX" redefined
 #define NOMINMAX

In file included from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/x86_64-w64-mingw32/bits/c++config.h:508,
                 from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/utility:68,
                 from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/algorithm:60,
                 from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:73,
                 from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/x86_64-w64-mingw32/bits/os_defines.h:45: note: this is the location of the previous definition
 #define NOMINMAX 1

In file included from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:329: warning: ignoring #pragma comment  [-Wunknown-pragmas]
 #pragma comment(lib, "psapi.lib")

C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:330: warning: ignoring #pragma comment  [-Wunknown-pragmas]
 #pragma comment(lib, "dbghelp.lib")

In file included from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:329: warning: ignoring #pragma comment  [-Wunknown-pragmas]
 #pragma comment(lib, "psapi.lib")

C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:330: warning: ignoring #pragma comment  [-Wunknown-pragmas]
 #pragma comment(lib, "dbghelp.lib")

In file included from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp: In member function 'backward::ResolvedTrace backward::TraceResolverImpl<backward::system_tag::windows_tag>::resolve(backward::ResolvedTrace)':
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:3356:32: warning: 'memset' used with constant zero length parameter; this could be due to transposed parameters [-Wmemset-transposed-args]
     memset(&sym, sizeof(sym), 0);
                                ^
In file included from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp: In member function 'backward::ResolvedTrace backward::TraceResolverImpl<backward::system_tag::windows_tag>::resolve(backward::ResolvedTrace)':
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:3356:32: warning: 'memset' used with constant zero length parameter; this could be due to transposed parameters [-Wmemset-transposed-args]
     memset(&sym, sizeof(sym), 0);
                                ^
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp: In constructor 'backward::SignalHandling::SignalHandling(const std::vector<int>&)':
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:4018:5: error: 'set_terminate' was not declared in this scope
     set_terminate(&terminator);
     ^~~~~~~~~~~~~
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:4018:5: note: suggested alternative:
In file included from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/new:40,
                 from
rd-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:
                 from  In constructor 'C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_tempbuf.h:60backward::SignalHandling::SignalHandling(const std::vector<int>&),
                 from ':
C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_algo.h:62C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:4018:5:,
                 from  C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/algorithm:62error: ,
                 from 'C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:73set_terminate,
                 from ' was not declared in this scope
     C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26set_terminate:
(&terminator);
     C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/exception:67:21:^~~~~~~~~~~~~
note:   'std::set_terminate'
   terminate_handler set_terminate(terminate_handler) _GLIBCXX_USE_NOEXCEPT;
                     C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:4018:5:^~~~~~~~~~~~~
In file included from note: C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26suggested alternative:
:
In file included from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:4019:5:C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/new:40 ,
                 from error: C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_construct.h:59',
                 from set_unexpectedC:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_tempbuf.h:60' was not declared in this scope
     ,
                 from set_unexpectedC:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_algo.h:62(&terminator);
     ,
                 from ^~~~~~~~~~~~~~C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/algorithm:62
,
                 from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:73,
                 from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:4019:5:C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26
note: C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/exception:67:21:suggested alternative:
In file included from  C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/new:40note: ,
                 from   'C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_construct.h:59std::set_terminate,
                 from '
   terminate_handler C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_tempbuf.h:60set_terminate,
                 from (terminate_handler) _GLIBCXX_USE_NOEXCEPT;
                     C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_algo.h:62^~~~~~~~~~~~~,
                 from
In file included from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/algorithm:62C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26,
                 from
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:73C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:4019:5:,
                 from  C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26error: :
 C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/exception:79:22:set_unexpected
     note: set_unexpected  '
     std::set_unexpected^~~~~~~~~~~~~~'
   unexpected_handler
set_unexpected(unexpected_handler) _GLIBCXX_USE_NOEXCEPT;
                      ^~~~~~~~~~~~~~
C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:4019:5: note: suggested alternative:
In file included from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/new:40,
                 from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_construct.h:59,
                 from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_tempbuf.h:60,
                 from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/bits/stl_algo.h:62,
                 from C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/algorithm:62,
                 from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.hpp:73,
                 from C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\source_subfolder\backward.cpp:26:
C:/Users/AndreiMalashkin/.conan/data/mingw-w64/8.1/_/_/package/6903a9d1b48b06f7fbe0929db654512c77e6cc32/lib/gcc/x86_64-w64-mingw32/8.1.0/include/c++/exception:79:22: note:   'std::set_unexpected'
   unexpected_handler set_unexpected(unexpected_handler) _GLIBCXX_USE_NOEXCEPT;
                      ^~~~~~~~~~~~~~
mingw32-make.exe[2]: *** [source_subfolder\CMakeFiles\backward_object.dir\build.make:76: source_subfolder/CMakeFiles/backward_object.dir/backward.cpp.obj] Error 1
mingw32-make.exe[1]: *** [CMakeFiles\Makefile2:99: source_subfolder/CMakeFiles/backward_object.dir/all] Error 2
mingw32-make.exe[1]: *** Waiting for unfinished jobs....
mingw32-make.exe[2]: *** [source_subfolder\CMakeFiles\backward.dir\build.make:76: source_subfolder/CMakeFiles/backward.dir/backward.cpp.obj] Error 1
mingw32-make.exe[1]: *** [CMakeFiles\Makefile2:125: source_subfolder/CMakeFiles/backward.dir/all] Error 2
mingw32-make.exe: *** [Makefile:135: all] Error 2
backward-cpp/1.5:
backward-cpp/1.5: ERROR: Package '4b8d03cfef6f209236f18445b773d36681be0d57' build failed
backward-cpp/1.5: WARN: Build folder C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57
ERROR: backward-cpp/1.5: Error in build() method, line 119
        cmake.build()
        ConanException: Error 2 while executing cmake --build C:\Users\AndreiMalashkin\.conan\data\backward-cpp\1.5\_\_\build\4b8d03cfef6f209236f18445b773d36681be0d57\build_subfolder -- -j12
```

</details>

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **backward-cpp/1.5**

Fixes #6661
+ modernize
+ double quotes
---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

File: recipes/backward-cpp/all/patches/backward-cpp-1.5-add-iterator-include.patch [hunk 1]
```diff
@@ -1,10 +1,10 @@
 --- backward.hpp.old	2020-05-05 20:37:23.473359490 +0200
```
Comment (prince-chrismc) [inline recipes/backward-cpp/all/patches/backward-cpp-1.5-add-iterator-include.patch:1]: 😆

**Review by prince-chrismc — APPROVED** [review]
Can you submit the patches upstream?

File: recipes/backward-cpp/all/patches/backward-cpp-1.5-add-iterator-include.patch (hunk 1 above)
Comment (madebr) [reply recipes/backward-cpp/all/patches/backward-cpp-1.5-add-iterator-include.patch:1]: This patch is already in upstream:
https://github.com/bombela/backward-cpp/blame/bd9389453a462c85a70f03718840ca0897c9088b/backward.hpp#L94
(3 months ago)

**Comment by madebr** [comment]
I'll (try to) upstream the mingw patch.
I've added it to a comment in their issue tracker, but a proper pr will be much more helpful.

**Comment by madebr** [comment]
See https://github.com/bombela/backward-cpp/pull/234

Looks like current master had already improved their mingw support a lot.

**Pull request merged**